@AH-64D Apache Longbow/Addons/fza_ah64_audioSystem/functions/fn_addASEMessage.sqf

```diff
@@ -0,0 +1,29 @@
+/* ----------------------------------------------------------------------------
+Function: fza_audioSystem_fnc_addASEMessage
+
+Description:
+
+Parameters:
+    _heli: object - the apache the player is in
+
+Returns:
+    No returns
+
+Examples:
+
+Author:
+    Rosd6(Dryden)
+---------------------------------------------------------------------------- */
+params ["_heli", "_priority", "_Reference"];
+
+private _aseMsg = _heli getVariable ["fza_audio_ase_message", ""];
+
+if (_aseMsg isEqualTo "" or {_priority > _aseMsg # 1}) then {
+    _heli setvariable ["fza_audio_ase_message", _this];
+};
+
+private _FuncHook = _heli getVariable ["fza_audio_FuncHook", scriptNull];
```

Comment:
This should be _funcHook, that way it follows the normal coding conventions already established in the mod.